### PR TITLE
Add post for SWIFT private preview

### PR DESCRIPTION
**Pull request by ruiminwang**
1. Add wiki for SWIFT private preview
2. Add issue templates

### CODE REVIEW

File: SWIFT-Private-Preview.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Azure Stream Analytics Private Preview
```
[the remark this answers is not in the document]
Comment (ruiminwang) [reply SWIFT-Private-Preview.md:1]: make sense!

> shouldn't this be VNET Support in Standard SKU Private Preview?

File: SWIFT-Private-Preview.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## VNET Support in Standard SKU
+
+## Important Information to Note 
+### Prerequisites:
+- Make sure your subscription has been whitelisted for the feature. If not, please create an [issue](https://github.com/microsoft/vscode/issues/new?assignees=&labels=&template=swift_prpr.md) here. Please wait until your issue is resolved. 
+- Available regions for private preview: **West US & North Europe**
+- Create or prepare your VNET: please note that your VNET and ASA job must be in the <u>**same region and same subscription**</u> for this private preview.
+
+### Gated preview success criteria:
```
[the remark this answers is not in the document]
Comment (ruiminwang) [reply SWIFT-Private-Preview.md:10]: > remove these criterias as we discussed

np~

File: SWIFT-Private-Preview.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    - There is a known scaling issue for jobs running with 1,3,6 SUs
+- When test connection and sample data right after creating a job, customer might encounter errors like below (just retry after one or two minutes)
+
+    ![connection test failed](./figures/SWIFT/connection_failed.png)
+
+
+## Setting up your Azure Stream Analytics Job
+1. [Install VSCode and ASA Tools extension](https://learn.microsoft.com/en-us/azure/stream-analytics/quick-create-visual-studio-code#prerequisites) if you haven’t done so already.
+
+2. Create your job in [VS Code](https://learn.microsoft.com/en-us/azure/stream-analytics/quick-create-visual-studio-code) as you would any other ASA job.
```
[the remark this answers is not in the document]
Comment (ruiminwang) [reply SWIFT-Private-Preview.md:36]: > similar to any other ASA job

how about `as you would do for any other ASA job`?

[the remark this answers is not in the document]
Comment (seanshi007) [reply SWIFT-Private-Preview.md:36]: sounds good!

File: .github/ISSUE_TEMPLATE/swift_prpr.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+name: SWIFT private preview
+about: Questions about VNET Support in Standard SKU
+title: "[SWIFT] Subscription whitelist"
+---
+
+<!-- ⚠️⚠️ Do Not Delete This! bug_report_template ⚠️⚠️ -->
+
+- VS Code Version:
```
[the remark this answers is not in the document]
Comment (ruiminwang) [reply .github/ISSUE_TEMPLATE/swift_prpr.md:9]: > might not need these right? Actually shall we mention somewhere (either in the wiki or here) that you must use extension starting from XXX to use SWIFT private preview feature.

We need these version for debugging. But we could also memtion they must use extension starting  from 1.4.3

[the remark this answers is not in the document]
Comment (seanshi007) [reply .github/ISSUE_TEMPLATE/swift_prpr.md:9]: This is here just for whitelist, so I think we shouldn't confuse them to provide these info. But it's your call.

**Review by seanshi007 — APPROVED** [review]

**Comment by ruiminwang** [comment]
> 

@microsoft-github-policy-service agree

**Pull request merged**